packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.Props.ts

```diff
@@ -0,0 +1,109 @@
+import { Position } from '../../utilities/positioning';
+import { IIconProps } from '../../Icon';
+
+export interface ISpinButtonProps {
+
+  /**
+   * The initial value of the SpinButton. Use this if you intend for the SpinButton to be an uncontrolled component.
+   * This value is mutually exclusive to value. Use one or the other.
+   * @default 0
+   */
+  defaultValue?: string;
+
+  /**
+   * The initial value of the SpinButton. Use this if you intend to pass in a new value as a result of onChange events.
+   * This value is mutually exclusive to defaultValue. Use one or the other.
+   */
+  value?: string;
+
+  /**
+   * The width of the SpinButton (including the optional label).
```

Comment:
This is a a little weird to me, shouldn't this be done via styling instead of props?